hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/OMMetadataManagerTestUtils.java

```diff
@@ -233,7 +234,7 @@ public static void writeKeyToOm(OMMetadataManager omMetadataManager,
                                   long bucketObjectId,
                                   long volumeObjectId,
                                   List<OmKeyLocationInfoGroup> locationVersions,
-                                  BucketLayout bucketLayout)
+                                  BucketLayout bucketLayout, long dataSize)
```

Comment:
let's keep it consistent, and move it to the next line.

Reply:
sure, updated!